src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests3.cs

```diff
@@ -1386,5 +1386,38 @@ public static (int?, int) Get(string name)
 ";
             CompileAndVerify(CreateCompilation(source, options: TestOptions.ReleaseExe).VerifyDiagnostics(), expectedOutput: "null");
         }
+
+        [Fact, WorkItem(38686, "https://github.com/dotnet/roslyn/issues/38686")]
+        public void TargetTypedSwitch_AnyTypedSwitchWithoutTargetType()
```

Comment:
>TargetTypedSwitch_AnyTypedSwitchWithoutTargetType [](start = 20, length = 49)

Please add corresponding IOperation tests to make sure that all constituent pieces are properly represented in the tree.

Reply:
Please add test for IOperation tree limited to the switch expression of at most the containing statement (see IOperationTests_SwitchExpression.SwitchExpression_Basic for example) as opposed to a control flow graph test.

---
In reply to: [326335381](https://github.com/dotnet/roslyn/pull/38750#discussion_r326335381) [](ancestors = 326335381)

Reply:
The IOperation test below (line 1548 and beyond) shows the IOperation tree for both switch expressions.  I am not going to modify it to test less than it already does.

---
In reply to: [326429634](https://github.com/dotnet/roslyn/pull/38750#discussion_r326429634) [](ancestors = 326429634,326335381)